lib/getJobsCompletedCount.ts

```diff
@@ -14,34 +11,22 @@ export default async (): Promise<number> => {
 }
 
 const updateCache = async (kv: Deno.Kv): Promise<number> => {
-  const jobsCompleted = await getFromInflux()
+  const jobsCompleted = await getFromApi()
   await kv.set(['jobs-completed'], jobsCompleted)
   return jobsCompleted
 }
 
-const OFFSET = 567_564_171
-const OFFSET_DATE = '2024-03-03T17:14:30.236298505Z'
+const OFFSET = 155_153_052_475
+const OFFSET_DATE = '2024-07-09'
 
-export const getFromInflux = async (): Promise<number> => {
-  const res = await fetch("https://eu-central-1-1.aws.cloud2.influxdata.com/api/v2/query", {
-    method: "POST",
-    headers: {
-      "Accept": "application/csv",
-      "Content-Type": "application/vnd.flux",
-      "Authorization": `Token ${Deno.env.get("INFLUX_TOKEN")}`,
-    },
-    body: `
-      from(bucket: "station")
-        |> range(start: ${OFFSET_DATE})
-        |> filter(fn: (r) => r["_measurement"] == "jobs-completed")
-        |> filter(fn: (r) => r["_field"] == "value")
-        |> group()
-        |> sum()
-    `,
-  })
-  const body = await res.text()
-  assert(res.ok, `Bad InfluxDB response: ${res.status} ${res.statusText}`)
-  return OFFSET + Number(
-    body.split('\n')[1].split(',')[5]
+export const getFromApi = async (): Promise<number> => {
+  console.log('getFromApi() ...')
+  const res = await fetch(`https://stats.filspark.com/measurements/daily?from=${OFFSET_DATE}`)
+  const days = await res.json()
+  const total = days.reduce(
+    (acc: number, day: any) => acc + day.accepted_measurement_count,
+    0
   )
+  console.log('getFromApi() ✔️')
+  return OFFSET + total
```

Comment:
How about including the count in the log message?

```suggestion
  const result = OFFSET + total
  console.log('getFromApi() counted %s jobs completed', result)
  return result
```

Reply:
I don't have a need for that 🤷  Just makes it more code